crates/walrus-service/src/test_utils.rs

```diff
@@ -1529,6 +1535,28 @@ impl CommitteeService for StubCommitteeService {
 pub struct StubContractService {
     pub(crate) system_parameters: FixedSystemParameters,
     pub(crate) node_capability_object: StorageNodeCap,
+    pub(crate) certify_event_blob_tx:
+        Arc<std::sync::Mutex<tokio::sync::mpsc::Sender<BlobObjectMetadata>>>,
+    #[allow(dead_code)]
+    pub(crate) certify_event_blob_rx:
+        Arc<std::sync::Mutex<tokio::sync::mpsc::Receiver<BlobObjectMetadata>>>,
```

Comment:
Suggestion: first of all, this is a great idea to test whether the node has done a certain things or not. I just wonder if we can make the test more explicit.

Right now, the test assumes that a SubContractService will be used in the test, and then use as_ayn to downcast it and check the channel. Instead, can we make it more explicit that in the test body, we build storage node with `with_committee_service()` (https://github.com/MystenLabs/walrus/blob/3788d74f2c783d195622d79fdfc25c7feae1bb60/crates/walrus-service/src/test_utils.rs#L746) that passing in a StubCommitteeService explicitly? This way, if the default behavior of using StubCommitteeService is changed, the test won't get affected. Also, we no longer need the as_any trait, which seems to be only used in tests. WDYT?